src/movies.js

```diff
@@ -1,16 +1,83 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(array){
+    const directors = array.map(function(el){
+      return el.director
```

Comment:
try to use more specific variable names so you can easily see what you're looking for in the future.